MAUI/DataGrid/data-binding.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Data Binding in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Data Binding support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.

```

Comment:
description: Learn all about Data Binding support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more here.

Reply:
Modified.